MapTour/src/app/storymaps/maptour/core/MainView.js

```diff
@@ -1,4 +1,5 @@
 define([
+    "./Maptiks",

```

Comment:
Maybe require this in line:27 of main-app.js? otherwise, we should comment why we're adding this here... (to load the Maptiks.js file)